test/unit/spec/v2/view-builder/internals/BaseModel_spec.js

```diff
@@ -259,28 +259,24 @@ describe('v2/view-builder/internals/BaseModel', function () {
             value: {
               id: 'autwa6eD9o02iBbtv0g3',
             },
-            authenticatorType: 'password',
```

Comment:
I might miss some detail. why this is not replaced with `authenticatorKey` ?

Reply:
I think we should, I missed this file...